inc/azure_macro_utils/macro_utils.h

```diff
@@ -78,73 +78,101 @@ MU_IF(X, "true", "false") => "true"
 #define MU_EAT_EMPTY_ARG(arg_count, arg) MU_IF(MU_ISEMPTY(arg),,arg) MU_IF(MU_ISEMPTY(arg),MU_EXPAND_TO_NOTHING,MU_IFCOMMALOGIC)(MU_DEC(arg_count))
 #define MU_EAT_EMPTY_ARGS(...) MU_FOR_EACH_1_COUNTED(MU_EAT_EMPTY_ARG, __VA_ARGS__)
 
+#define MU_EMPTY()
+#define MACRO_UTILS_DELAY(id) id MU_EMPTY MU_LPAREN )
+
+//
+//
+// MU_DEFINE_ENUM
+//
+//
+
+// This uber enum macro can handle enumeration-constants being constant expressions.
```

Comment:
>uber [](start = 8, length = 4)

qualification unnecessary, should be obvious from itself.

Reply:
Removed.

---
In reply to: [328346554](https://github.com/Azure/azure-macro-utils-c/pull/30#discussion_r328346554) [](ancestors = 328346554)